### PR TITLE
OCPBUGS-6070#Fix parameter in user-encryption for AWS EBS

### PR DESCRIPTION
**Pull request by lpettyjo**
Version(s): 4.13+

Issue: https://issues.redhat.com/browse/OSDOCS-6070

Link to docs preview: https://79855--ocpdocs-pr.netlify.app/openshift-enterprise/latest/storage/container_storage_interface/persistent-storage-csi-ebs.html#byok_persistent-storage-csi-ebs

QE review:
- [ ] QE has approved this change.

Additional information:

**PTAL**: @dobsonj @Phaow

### CODE REVIEW

File: modules/persistent-storage-byok.adoc [hunk 1]
```diff
@@ -19,3 +19,19 @@ This features supports the following storage types:
 * Google Cloud Platform (GCP) persistent disk (PD) storage
 
 * IBM Virtual Private Cloud (VPC) Block storage
+
+You must specify the custom key in the appropriate field in the install-config YAML file as specified in the following table.
```
Comment (cbippley) [inline modules/persistent-storage-byok.adoc:23]:
```suggestion
You must specify the custom key in the appropriate field in the `install-config.yaml` file as specified in the following table.
```
I'm seeing it presented this way in the docs.

**Review by cbippley — COMMENTED** [review]
One nit, LGTM! QE review is needed before merge review.

**Comment by Phaow** [comment]
/hold
Origin doc is correct, this should not a bug. Added comment in the bug ->
https://issues.redhat.com/browse/OSDOCS-6070?focusedId=25251832&page=com.atlassian.jira.plugin.system.issuetabpanels:comment-tabpanel#comment-25251832

**Comment by lpettyjo** [comment]
Closing PR since there is actually no doc issue.

**Pull request closed without merging**